site/public/js/server.js

```diff
@@ -1,14 +1,5 @@
-var csrfToken = undefined;
```

Comment:
is this edit related to this PR?

Reply:
Not strictly needed, but as I was adding a new variable to be loaded into `window`, I took the opportunity to clean up how these were being set (the `var csrfToken = undefined` is technically doing `window.csrfToken = undefiend`) so that they're not available immediately on page load, instead of having to wait for the onload function below that line to run.